state-chain/traits/src/mocks/bonding.rs

```diff
@@ -0,0 +1,43 @@
+use crate::Bonding;
+use std::{cell::RefCell, collections::HashMap};
+
+use crate::DispatchResult;
+
+pub type Amount = u128;
+pub type ValidatorId = u64;
+
+thread_local! {
+	pub static AUTHORITY_BONDS: RefCell<HashMap<ValidatorId, Amount>> = RefCell::new(HashMap::default());
+}
```

Comment:
Prefer to use storage otherwise you might see some strange side effects (for example, thread_locals are not transactional so they don't obey `assert_noop`).